services/IoT/applications/libraries/java.md

```diff
@@ -234,7 +234,7 @@ Overloaded methods are available to control the command subscription. The proces
 ## Subscribing to device status
 {: #subscribing_device_status}
 
-Similar to subscribing to device events, applications can subscribe to device status, like device connect and disconnect to {{site.data.keyword.iot_short_notm}}. By default, this will subscribe to status updates for all connected devices. Use the Device Type and Device ID parameters to control the scope of the subscription. A single ApplicationClient can support multiple subscriptions.
+Similar to subscribing to device events, applications can subscribe to device status, such as device connect and disconnect to {{site.data.keyword.iot_short_notm}}. By default, this subscription will subscribe to status updates for all connected devices. Use the Device Type and Device ID parameters to control the scope of the subscription. A single ApplicationClient can support multiple subscriptions.
 
 ### Subscribe to status updates for all devices
 
```

Comment:
Line 281
"The following properties are only set when the action is "Disconnect":" - but line 270 says  that the propreties are set for "both "Connect" and "Disconnect" status events" - so what are these, actions or status events? need to standardize terminology and use consistently.